aws_lambda.py

```diff
@@ -414,6 +411,24 @@ def sync_to_s3(target_dir, aws_region=AWS_REGION, bucket_name=BUCKET_NAME):
             aws_region, bucket_name, filename))
 
 
+def help(event, context):
+    """Show this help.
+    """
+    def white_bold(s):
+        return f"\033[1m\x1B[37m{s}\033[0;0m"
+
+    commands = [obj for name, obj in inspect.getmembers(sys.modules[__name__])
+                if inspect.isfunction(obj) and not name.startswith("_")]
```

Comment:
This is clever -- maybe too clever. First off, instead of rewriting a bunch of imports, we could have just checked `inspect.getmodule(thing) == sys.modules[__name__]`. But even so, I'm nervous about someone adding a function that doesn't start with an underscore, not noticing this functionality. (I only noticed it after the third time reviewing it. I was about to assume you were just doing some kind of import cleanup for some other reason, maybe `black`.) Could we maybe add a decorator instead to explicitly flag entry points? After all "explicit is better than implicit".